examples/hybrid/plot_scaling_results.jl

```diff
@@ -68,113 +61,147 @@ for foldername in readdir(output_dir)
+    savefig(figpath)
+end
 
-figpath = joinpath(output_dir, resolution * "_" * "Scaling.png")
 
 Plots.GRBackend()
-Plots.png(
-    plot(
-        [nprocs nprocs_tempest],
-        [cpu_hours cpu_hours_tempest],
-        markershape = [:circle :square],
-        xticks = (nprocs, [string(i) for i in nprocs]),
-        xlabel = "nprocs",
-        ylabel = "CPU hours",
-        title = "Scaling data (integration time = " * t_int * ")",
-        label = ["ClimaAtmos (Float32)" "Tempest (Float64)"],
-        legend = :topleft,
-        grid = :true,
-        left_margin = 10mm,
-        bottom_margin = 10mm,
-        top_margin = 10mm,
+plot(
+    log2.(nprocs_clima_atmos),
+    cpu_hours_clima_atmos,
+    markershape = :circle,
+    markercolor = :blue,
+    xticks = (
+        log2.(nprocs_clima_atmos),
+        [string(i) for i in nprocs_clima_atmos],
     ),
-    figpath,
+    xlabel = "nprocs",
+    ylabel = "CPU hours",
+    title = "Scaling data (integration time = $t_int)",
+    label = "ClimaAtmos (Float32)",
+)
+plot!(
+    log2.(nprocs_tempest),
+    cpu_hours_tempest,
+    markershape = :square,
+    markercolor = :orange,
+    label = "Tempest (Float64)",
+    legend = :topleft,
+    grid = :true,
+    left_margin = 10mm,
+    bottom_margin = 10mm,
+    top_margin = 10mm,
 )
 
-linkfig(relpath(figpath, joinpath(@__DIR__, "../")), "Scaling Data")
+for figtype in figtypes
+    figpath = joinpath(output_dir, resolution * "_" * "Scaling$figtype")
+    savefig(figpath)
+end
 
-figpath = joinpath(output_dir, resolution * "_" * "Scaling_efficiency.png")
 
 Plots.GRBackend()
-Plots.png(
-    plot(
-        [nprocs nprocs_tempest],
-        [scaling_efficiency scaling_efficiency_tempest],
-        markershape = [:circle :square],
-        xticks = (nprocs, [string(i) for i in nprocs]),
-        ylims = ((0, 110)),
-        xlabel = "nprocs",
-        ylabel = "Efficiency",
-        title = "Scaling efficiency",
-        label = ["ClimaAtmos (Float32)" "Tempest (Float64)"],
-        legend = :bottom,
-        grid = :true,
-        left_margin = 10mm,
-        bottom_margin = 10mm,
-        top_margin = 10mm,
-        annotations = [
-            (
-                nprocs,
-                scaling_efficiency .- 5,
-                [string(i) * "%" for i in scaling_efficiency],
-                4,
-            ),
-            (
-                nprocs_tempest,
-                scaling_efficiency_tempest .- 5,
-                [string(i) * "%" for i in scaling_efficiency_tempest],
-                4,
-            ),
-        ],
+plot(
+    log2.(nprocs_clima_atmos),
+    scaling_efficiency_clima_atmos,
+    markershape = :circle,
+    markercolor = :blue,
+    xticks = (
+        log2.(nprocs_clima_atmos),
+        [string(i) for i in nprocs_clima_atmos],
     ),
-    figpath,
+    xlabel = "nprocs",
+    ylabel = "Efficiency",
+    title = "Scaling efficiency (integration time = $t_int)",
```

Comment:
- Could you please include the eq. that is used to calculate efficiency in the axes labels ( might help clarify results at a glance; I think this corresponds to lines 82-88 or so in this code.) 
- Some of the axis ticks / plot labels were overlapping (based on results shown this morning), as long as those are legible the plots seemed fine.

Reply:
(Also perhaps a quick reference to the >100% values and how we interpret those)

Reply:
For scaling efficiency, we are using 
```
eff = (T1/N)/TN where T1 -> simulation time with single MPI process; TN -> simulation with N MPI processes; N -> # of MPI processes
```